app/controllers/medicaid/amounts_expenses_controller.rb

```diff
@@ -6,8 +6,8 @@ class AmountsExpensesController < StandardStepsController
 
     private
 
-    def ensure_application_present
-      true
+    def step_class
```

Comment:
I wonder if we should add some kind of shared example for these controllers to ensure they all have this private method? Is that too invasive?

Reply:
we do have one in `app/controllers/steps_controller.rb` but it doesn't work for plural step class names because it calls `controller_path.classify.constantize`